src/ReactiveDomain.UI.Testing/TestVisibilityHelpers.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Reflection;
+
+namespace ReactiveDomain.UI.Testing
+{
+    public static class TestVisibilityExtensions
+    {
+        public static T CurrentValue<T>(this IObservable<T> readModelProperty)
+        {
+            if (!(readModelProperty is ReadModelProperty<T>)) { new InvalidOperationException($"Observable is not a ReadModelProperty."); }
+            return GetInstanceField<T>(readModelProperty, "_lastValue");
+        }
+        internal static T GetInstanceField<T>(object instance, string fieldName)
```

Comment:
Instead of duplicating this from `ReactiveDomain.Testing`, can we make that other project's internals visible to `ReactiveDomain.UI.Testing`?

Reply:
the dependcies are a mess due to the windows only, I think a bit of duplication for boiler plate helpers isn't a problem.